src/main/java/org/dependencytrack/tasks/vulnerabilitypolicy/S3Client.java

```diff
@@ -0,0 +1,57 @@
+package org.dependencytrack.tasks.vulnerabilitypolicy;
+
+import alpine.Config;
+import alpine.common.logging.Logger;
+import io.minio.MinioClient;
+import org.dependencytrack.common.ConfigKey;
+
+public class S3Client {
+    private final String bundleToFetch;
+    private final String s3AccessKey;
+    private final String s3SecretKey;
+    private final String s3BucketName;
+    private MinioClient minioClient;
+
+    private String bundleFetchEndpoint;
+
+    private final String region;
+
+    public String getBundleToFetch() {
+        return bundleToFetch;
+    }
+
+    public String getS3BucketName() {
+        return s3BucketName;
+    }
+
+    private static final Logger LOGGER = Logger.getLogger(S3Client.class);
+
+    public S3Client() {
+        this.bundleFetchEndpoint = Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_BUNDLE_URL);
+        this.bundleToFetch = Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_BUNDLE_NAME);
+        this.s3AccessKey = Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_ACCESS_KEY);
+        this.s3SecretKey = Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_SECRET_KEY);
+        this.s3BucketName = Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_BUCKET_NAME);
+        this.region = Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_REGION);
```

Comment:
I think this class would be easier to test if we'd use a package-private constructor instead, which accepts all of these values as arguments. In the test, we could then simply construct instances with:

```java
new S3Client(myBundleFetchEndpoint, myBundleToFetch, myAccessKey, mySecretKey, myBucketName, myRegion);
```

For the "production" side of things, where the default `public` constructor is used, you can simply do:

```java
public S3Client() {
    this(Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_BUNDLE_URL),
        Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_BUNDLE_NAME),
        Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_ACCESS_KEY),
        Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_SECRET_KEY),
        Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_BUCKET_NAME),
        Config.getInstance().getProperty(ConfigKey.VULNERABILITY_POLICY_S3_REGION));
}
```

Fiddling with environment variables in a unit test should be a workaround, not the norm.